detection_rules/kbwrap.py

```diff
@@ -157,14 +178,20 @@ def kibana_export_rules(
         rules_count = results[-1]["exported_rules_count"]
         exception_list_count = results[-1]["exported_exception_list_count"]
         exception_list_item_count = results[-1]["exported_exception_list_item_count"]
+        action_connector_count = results[-1]["exported_action_connector_count"]
```

Comment:
remind me, what's the `[-1]`?

Reply:
[-1] is to get the last entry in results. Kibana returns a summary of what is included in the prior return so we can use that summary information to know more directly what part of the message deals with rule information, action information, and exception information respectively.